python/sbp/client/loggers/rotating_logger.py

```diff
@@ -11,7 +11,11 @@
 from logging.handlers import TimedRotatingFileHandler
 from sbp.client.loggers.json_logger import JSONLogger
 from sbp.table import dispatch
-import Queue
+import six
```

Comment:
Replace with `from six.moves.queue import Queue`.

Reply:
this file doesn't actually seem to use queue at all

Reply:
👍